Katello/katello
# Fixes #34659 - Change Katello bookmark to Foremans

#### What are the changes introduced in this pull request?

Maria said should change the Katello bookmark to use Foremans icon

#### Considerations taken when implementing this change?

Removed usage of the `<i className="fas fa-bookmark" />`

#### What are the testing steps for this pull request?

* Check out PR
* Visit any of the new host detail pages where there is content like packages/errata
* Verify the book mark is the new one
